front/src/modules/timesStops/hooks/useOutputTableData.ts

```diff
@@ -4,154 +4,124 @@ import { keyBy } from 'lodash';
 import { useTranslation } from 'react-i18next';
 
 import type {
-  OperationalPointWithTimeAndSpeed,
   PathPropertiesFormatted,
   SimulationResponseSuccess,
 } from 'applications/operationalStudies/types';
 import type { PathfindingResultSuccess, TrainScheduleResult } from 'common/api/osrdEditoastApi';
-import { formatSuggestedOperationalPoints } from 'modules/pathfinding/utils';
-import type { SuggestedOP } from 'modules/trainschedule/components/ManageTrainSchedule/types';
-import { convertIsoUtcToLocalTime } from 'utils/date';
-import { secToHoursString } from 'utils/timeManipulation';
+import { interpolateValue } from 'modules/simulationResult/SimulationResultExport/utils';
+import type { TrainScheduleWithDetails } from 'modules/trainschedule/components/Timetable/types';
+import { dateToHHMM } from 'utils/date';
 
-import { checkAndFormatCalculatedArrival } from '../helpers/arrivalTime';
+import { computeInputDatetimes } from '../helpers/arrivalTime';
 import computeMargins from '../helpers/computeMargins';
-import { computeScheduleData, formatScheduleData } from '../helpers/scheduleData';
-import {
-  findNextScheduledOpPoint,
```

Comment:
We can remove this function from utils and its tests

Reply:
Done